plugins/flashrom/fu-flashrom-lspcon-i2c-spi-device.c

```diff
@@ -328,6 +409,9 @@ fu_flashrom_lspcon_i2c_spi_device_write_firmware (FuDevice *device,
 		return FALSE;
 	}
 
+	/* device (re)loads its firmware at power-on */
+	fu_device_add_flag (device, FWUPD_DEVICE_FLAG_NEEDS_SHUTDOWN);
```

Comment:
Does warm reboot help it too?  Or it really needs _shutdown_?

Reply:
It really needs shutdown, determined by experiment.